src/components/AudioTrack/PlayPause.vue

```diff
@@ -43,13 +48,16 @@ export default {
     },
   }),
   computed: {
+    isPlaying() {
+      return this.status === 'playing'
+    },
     label() {
       return this.$t(this.isPlaying ? 'play-pause.pause' : 'play-pause.play')
     },
   },
   methods: {
-    toggle() {
-      this.$emit('toggle', !this.isPlaying)
+    handleClick() {
+      this.$emit('toggle', this.isPlaying ? 'paused' : 'playing')
```

Comment:
Nice, sending an enum is much clearer here.

Reply:
Yeah, boolean was ambiguous. Plus we can expand to more granular statuses like loading, errored etc. in the future.